lib/resources/orgs.js

```diff
@@ -21,6 +21,19 @@ class Orgs {
   find (organization) {
     return this.transport.request(`/orgs/${organization}`);
   }
+
+  /**
+   * List organization's members
+   *
+   * @param {string} organizatio
+   *
+   * @return {Promise}
+   *
+   */
+  listMembers (organization) {
```

Comment:
Technically you're not "listing" the members. I'd say something like findMembers() or getMembers().